api/auth.yml

```diff
@@ -542,6 +553,31 @@ components:
           uniqueItems: true
           items:
             type: string
+    HealthInfo:
```

Comment:
@manuio Is it possible to extract `HealthInfo` to a separate file and share it between services?

Reply:
@dusanb94 done. The only problem is if for example you want to verify yaml format with the online swagger editor, the example will be empty.

Reply:
We are also loosing specific service description...